osu.Game/Screens/Play/ReplayPlayer.cs

```diff
@@ -92,16 +96,22 @@ protected override void PrepareReplay()
 
         public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
-            const double keyboard_seek_amount = 5000;
-
             switch (e.Action)
             {
+                case GlobalAction.StepReplayBackward:
+                    StepFrame(-1);
+                    return true;
+
+                case GlobalAction.StepReplayForward:
+                    StepFrame(1);
+                    return true;
+
                 case GlobalAction.SeekReplayBackward:
-                    keyboardSeek(-1);
+                    SeekInDirection(-1);
```

Comment:
Is there any particular reason why this seeks in second increments rather than 5-second increments like previously?

Reply:
Fair enough